qa/tasks/cephfs/cephfs_test_case.py

```diff
@@ -243,8 +243,8 @@ def assert_session_state(self, client_id,  expected_state):
     def get_session_data(self, client_id):
         return self._session_by_id(client_id)
 
-    def _session_list(self):
-        ls_data = self.fs.mds_asok(['session', 'ls'])
+    def _session_list(self, rank=None, status=None):
+        ls_data = self.fs.rank_asok(['session', 'ls'], rank=rank, status=status)
```

Comment:
You can just add `rank` param and use it with the used helper i.e. `mds_asok`, it also accepts ranks (both `mds_asok` and `rank_asok` call `json_asok` at the end). My intention here is to make minimal changes.

Reply:
If you see, `mds_asok` is used in some(majority) functions, while `rank_asok` in some(like `perf_dump`). I'm not quite sure if this change is actually needed.

Reply:
`rank_asok` is preferred, generally. I think this change is good as-is.